dev/com.ibm.ws.webcontainer.servlet.4.0_fat/publish/files/serverConfigs/EnablePostOnlyJSecurityCheckServer.xml

```diff
@@ -0,0 +1,22 @@
+<server description="Extended Server for testing Servlet 50 jakarta.servlet">
+
+  <include location="../fatTestPorts.xml"/>
+
+  <featureManager>
+    <feature>servlet-5.0</feature>
+    <feature>jsp-2.3</feature>
+    <feature>appSecurity-2.0</feature>
+  </featureManager>
+
+  <webContainer enablePostOnlyJSecurityCheck="false"/> 
+
+  <logging
+    traceSpecification="*=info=enabled:com.ibm.ws.webcontainer*=all:com.ibm.wsspi.webcontainer*=all:GenericBNF=all"
+    maxFileSize="20"
+    maxFiles="10"
+    traceFormat="BASIC"/>
+    
+    <!--Java2 security-->
+  <javaPermission className="java.security.AllPermission"  name="*" actions="*" />
```

Comment:
Is this permission actually necessary?

Reply:
yes, I got a WARNING about Java 2 permission without it.